src/Sylius/Behat/Service/JQueryHelper.php

```
@@ -20,13 +20,13 @@ abstract class JQueryHelper
 {
     public static function waitForAsynchronousActionsToFinish(Session $session): void
     {
-        $session->wait(1000, "!document.querySelector('[data-live-loading=true]')");
+        $session->wait(1000, "!document.querySelector('[data-live-is-loading]')");
     }
 
     public static function waitForFormToStopLoading(DocumentElement $document, int $timeout = 10): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$timeout'. (undefined)

(UnusedFormalParameter)